dumb-jump.el

```diff
@@ -1918,16 +1922,9 @@ Optionally pass t for RUN-NOT-TESTS to see a list of all failed rules"
 
 (defun dumb-jump-get-point-context (line func cur-pos)
   "Get the LINE context to the left and right of FUNC using CUR-POS as hint."
-  (let* ((loc (dumb-jump-find-start-pos line func cur-pos))
-         (func-len (length func))
-         (sen-len (length line))
-         (right-loc-start (+ loc func-len))
-         (right-loc-end (length line))
-         (left (substring line 0 loc))
-         (right (if (> right-loc-end sen-len)
-                    ""
-                  (substring line right-loc-start right-loc-end))))
-    `(:left ,left :right ,right)))
+  (let* ((loc (or (cl-search func line :start2 cur-pos) 0)))
```

Comment:
Currently, we only attempt to load `cl-generic`, but not `cl-lib` or `cl-seq`. Since we're already using a few cl-lib functions, we should probably require `cl-lib` anyway.